Sources/Valkey/Connection/ValkeyConnectionConfiguration.swift

```diff
@@ -121,6 +121,9 @@ public struct ValkeyConnectionConfiguration: Sendable {
     /// Readonly connections can run readonly commands on replica nodes
     public var readOnly: Bool
 
+    // The name of Valkey Database
```

Comment:
If you use a triple forward slash, the documentation generation will pick up this comment 
```suggestion
    /// The number of Valkey Database
```